src/services/openid.ts

```diff
@@ -394,8 +396,24 @@ export class OpenIdService {
             )
         ).pipe(shareReplay(1));
       } else {
+        const url = new URL(authUri);
+        const allowInsecure =
+          url.hostname === "localhost" &&
+          url.protocol === "http:" &&
+          clientId === "local-anode-client";
+
+        const discoveryOptions = allowInsecure
+          ? { execute: [this.client.allowInsecureRequests] }
```

Comment:
This is needed to allow openid-client to connect to http://localhost:8787 (rather than https)